src/Basics.elm

```diff
@@ -612,9 +612,21 @@ e =
 -- ANGLES
 
 
-{-| Convert radians to standard Elm angles (radians).
+{-| Represent radians as standard Elm angles (radians).
 
     radians pi == 3.141592653589793
+
+    **Note:** Unlike the other angle functions, [`radians`](#radians) does not
+perform a conversion since the output, a `Float` representing radians, is the
+same as the input. Despite merely returning its input, the [`radians`](#radians)
+function is still useful: it adds semantics to angle values. For example:
+
+    angle = radians 3.14
```

Comment:
there should be an or between the two somewhere as with the current wording it is implied that they are a single statement. (Yes, anyone who can think gets what it means but it would still be better with an or)